rollup.config.js

```diff
@@ -8,14 +8,26 @@ export default [
       name: 'monads',
       file: `dist/${pkg.main}`,
       format: 'umd',
+      globals: {
+        rxjs: 'rxjs',
+        'rxjs/operators': 'rxjs/operators'
```

Comment:
Kind of wondering if this is a valid way to provide the global symbol for rxjs operators, or if it should be shortened to `operators` or something.

Reply:
Results in this in the UMD output:

```
(global = global || self, factory(global.monads = {}, global.rxjs, global['rxjs/operators']));
```

Reply:
I don't know the answer to this one. I think the way you have it is good.